Code Quality/Creating a pull request.md

```diff
@@ -0,0 +1,35 @@
+> If the reviews hurt they're probably right on some level. - Sean Lennon
+
+In the Infinum JavaScript team we love code reviews. We love getting them and giving them. This document aims to show how a good pull request should look like (no matter what SCM solution or tool). The main idea is to give your reviewer as much context as possible to allow them to focus on the feature or fix you have delivered.
+
+## Creating a pull request
+
+Your pull request should have **all of the below**:
+
+-   Task ID (Productive, Jira, etc.)
+-   A title
+-   A description that includes
+    -   links to the design files that you used
+    -   relevant links to documentation outlining your work
+    -   a screen shoot of a feature or fix you delivered
+
+The following are extra but highly recommended:
+
+-   a summary of changes and reasoning behind them
```

Comment:
I found many conflicting sources about whether bullet lists should begin with a capital letter or not and how they should be punctuated. It seems to vary in different cases and is partially up to the author to decide. Whatever the case, I think we should at least be consistent. The bullet list above has a mix of upper an lower case, this list is lower case and the list below is upper case.

Reply:
Fixed.